target/linux/mediatek/filogic/base-files/etc/init.d/bootcount

```diff
@@ -5,6 +5,24 @@ START=99
 
 boot() {
 	case $(board_name) in
+	xiaomi,mi-router-ax3000t)
+		. /lib/upgrade/common.sh
+		[ "$(rootfs_type)" = "tmpfs" ] && \
+			logger "bootcount: initramfs mode detected, exit" && \
+			return 0
+		[ $(fw_printenv -n flag_try_sys2_failed) -eq 8 ] && \
```

Comment:
hmm, prefer use `=` instead of `-eq`
and `""` is needed.

Reply:
Why?

Reply:
I take your advice. Additionally, we should avoid error if `flag_try_sys2_failed` isn't exist. So, let's redirect `stderr`:
`[ "$(fw_printenv -n flag_try_sys2_failed 2>&1)" = "8" ] && \`